python/lsst/pipe/base/graphBuilder.py

```diff
@@ -381,7 +399,10 @@ def __repr__(self) -> str:
     quantum.
     """
 
-    def makeQuantum(self, datastore_records: Optional[Mapping[str, DatastoreRecordData]] = None) -> Quantum:
+    def makeQuantum(
```

Comment:
Why did this get reformatted? I can't see what changed. Was it reformatted because of an early API modification that was backed out?

Reply:
Yes, that's exactly it.  Black vs. the trailing commas.